.github/workflows/ci.yml

```diff
@@ -146,7 +146,7 @@ jobs:
         env:
```

Comment:
`php_unit_data_provider_static` was added smoothly only because the test was checking migrations with only 2 digits, so `PHPUnit100Migration:risky` was not tested for having fixers with `target` option.

I know what `target` does, but I don't know if that requirement of making it mandatory has some more depth.